torchvision/models/_api.py

```diff
@@ -1,19 +1,52 @@
+from __future__ import annotations
+
 import importlib
 import inspect
 import sys
 from dataclasses import dataclass, fields
 from inspect import signature
 from types import ModuleType
-from typing import Any, Callable, cast, Dict, List, Mapping, Optional, TypeVar, Union
+from typing import Any, Callable, cast, Dict, List, Mapping, Optional, Tuple, TypeVar, Union
 
 from torch import nn
 
 from torchvision._utils import StrEnum
 
 from .._internally_replaced_utils import load_state_dict_from_url
 
-
-__all__ = ["WeightsEnum", "Weights", "get_model", "get_model_builder", "get_model_weights", "get_weight", "list_models"]
+__all__ = [
+    "TransformsFactory",
```

Comment:
I assume we do not need this to be public? And if it's not public, does it need to be in `__all__`?

Reply:
Right now we don't. But with #7094 in mind, this is probably a useful helper for users as well. No strong opinion though.